src/reformat.c

```diff
@@ -14,6 +14,15 @@
 #include <pthread.h>
 #endif
 
+static void * avifMemset16(void * dest, int val, size_t count)
+{
+    size_t i;
```

Comment:
C99 is OK here, you can scope this to the loop.

Reply:
Done.